src/libraries/System.Security.Cryptography.Csp/src/Resources/Strings.resx

```diff
@@ -1,16 +1,16 @@
 ﻿<root>
-  <!-- 
-    Microsoft ResX Schema 
-    
+  <!--
```

Comment:
Sorry for the noise here, can undo if its undesirable in this PR, I'm assuming some editor config setting is removing the excessive white space.